frontend/packages/knative-plugin/integration-tests/features/eventing/kafka-event-sink.feature

```diff
@@ -23,10 +23,50 @@ Feature: Kafka Event Sink
              Then user will see "kafka-sink-test3" created in topology
         
         @regression @odc-6741
-        Scenario: Delete kafka sink: KN-01-TC02
-            Given user is at Topology page
+        Scenario: Add kafka sink as Trigger to Broker: KS-01-TC02
+            Given user has created Broker "broker-test1" in topology
+              And user has created KafkaSink "kafka-sink-test3" in topology
+              And user is at Topology page
+              And user is at Topology Graph view
+             When user right clicks on the Broker "broker-test1" to open the context menu 
+              And user clicks on the Add Trigger
+              And user selects the auto populated name of subscription
+              And user will click on the Subscriber dropdown on the modal
+              And user selects the Subscriber "kafka-sink-test3"
+              And user clicks on Add button
+             Then user will see connection between Broker and Subscriber
+
+         @regression @odc-6741
+         Scenario: Add kafka sink as Subscription to channel: KS-01-TC03
+            Given user has created Channel "channel-test1" in topology
+              And user has created KafkaSink "kafka-sink-test3" in topology
+              And user is at Topology page
               And user is at Topology Graph view
+             When user right clicks on the Channel "channel-test1" to open the context menu
+              And user selects "Add Subscription" from Context Menu
+              And user enters Name as "channel-subscrip" on Add Subscription modal
+              And user selects Subscriber "kafka-sink-test3" on Add Subscription modal
+              And user clicks on Add button
+             Then user will see connection between Channel "channel-test1" and Subscriber "kafka-sink-test3"
+
+        @regression @odc-6741
+        Scenario: Sink Event Source with existing kafka sink: KE-01-TC04
+            Given user has created knative service "nodejs-ex-git"
+              And user has created Sink Binding event source "sink-binding" with knative resource "nodejs-ex-git"
               And user has created KafkaSink "kafka-sink-test3" in topology
-             When user selects "Delete KafkaSink" context menu option of kafka sink "kafka-sink-test3"
-              And user clicks Delete button on Delete modal
-             Then user will not see "kafka-sink-test3" in topology
+              And user is at Topology page
+             When user right clicks on the event source "sink-binding" to open context menu
+              And user selects "Move sink" from context menu
+              And user selects the Resource "kafka-sink-test3" in "Move sink" modal
+              And user clicks on save
+             Then user will see that event source "sink-binding" is sinked with kafka sink "nodejs-ex-git-1"
```

Comment:
This would be `kafka-sink-test3` as you have moved connection to resource of kafka sink
```suggestion
             Then user will see that event source "sink-binding" is sinked with kafka sink "kafka-sink-test3"
```